docs/sources/companion-plugins/dropbox.mdx

```diff
@@ -127,6 +127,9 @@ Things to note:
 - Choose the “Dropbox API”, not the business variant.
 - Typically you’ll want “Full Dropbox” access, unless you are absolutely certain
   that you need the other one.
+- If you want to support
+  [Team folders / Teams spaces](https://developers.dropbox.com/dbx-team-files-guide),
+  counterintuitively you must **not** enable any Team scopes under Permissions.
```

Comment:
Shouldn't we say then what you should do? The phrasing makes it sound like something is missing?

Reply:
good point. i've updated and added instructions aobut which scopes are needed now